ml-testing-toolkit/chart-backend/templates/statefulset.yaml

```diff
@@ -63,6 +63,55 @@ spec:
         {{- if .Values.initContainers }}
           {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
         {{- end }}
+        - name: copy-original-files
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}          
+          command:
+            - sh
+            - -c
+            - |
+              set -e
+              echo "Copying files to /tmp/stage"
+              mkdir -p /tmp/stage
+              cp -r -v /opt/app/spec_files/. /tmp/stage/ || { echo "Copy to /tmp/stage failed"; exit 1; }
+              echo "Setting permissions in /tmp/stage:"
+              chmod -R 644 /tmp/stage/* 2>/dev/null || true
+              find /tmp/stage -type d -exec chmod 755 {} \; 2>/dev/null || true
+              echo "Copying from /tmp/stage to /target"
+              cp -r -v /tmp/stage/. /target/ || { echo "Copy to /target failed"; exit 1; }
+          volumeMounts:
+            - name: spec-files
+              mountPath: /target
+          resources: {}
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+        - name: add-extra-specs
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - sh
+            - -c
+            - |
+              set -e
+              cd /opt/default_config
+              for FILE in *; do
+                CONTENT=$(cat "$FILE")
+                DEST="/opt/app/spec_files/${FILE//__/\/}"
+                mkdir -p "$(dirname "$DEST")"
+                if echo "$CONTENT" | grep -q '^"http'; then
+                  URL=$(echo "$CONTENT" | tr -d '"')
+                  echo "Downloading $URL to $DEST"
+                  wget -q -O "$DEST" "$URL"
```

Comment:
Please note that recent version of TTK backend can download the config files on its own, so there is no need this to happen in the chart. We are trying to run all containers with read only file system to improve security and ensure stateless services.
See https://github.com/mojaloop/ml-testing-toolkit/pull/332

Reply:
hi @kalinkrustev are there any config changes required for this to work? At what stage it downloads the files?  If the root filesystem is readonly will the download work? The current mojaloop/values.yaml in the helm repo doesn't work with your latest change. The files are not downloaded.